dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityServlet.java

```diff
@@ -67,7 +80,7 @@ protected final void service(HttpServletRequest req, HttpServletResponse respons
         final VelocityRequestWrapper request =VelocityRequestWrapper.wrapVelocityRequest(req);
         final String uri = CMSUrlUtil.getCurrentURI(request);
         final boolean comeFromSomeWhere = request.getHeader("referer") != null;
```

Comment:
I think 'comeFromSomeWhere' is not need anymore, right?

Reply:
dunno man.